src/strategies/sunrisegaming-univ2-lp/index.ts

```diff
@@ -0,0 +1,179 @@
+import { multicall } from '../../utils';
+import { formatUnits } from '@ethersproject/units';
+import { BigNumber } from '@ethersproject/bignumber';
+
+export const author = 'sunrisegaming';
```

Comment:
should be a github username
```suggestion
export const author = 'sunrisedao';
```